src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -115,7 +115,7 @@ private constructor(
                       version = ConfigUtil.getPluginVersion(),
                       ideVersion = ApplicationInfo.getInstance().build.toString(),
                       workspaceRootUri =
-                          ProtocolTextDocument.normalizeUriOrPath(
+                          ProtocolTextDocumentUtil.normalizeUriOrPath(
```

Comment:
Why not put it in the `protocol_extensions` and use the existing extension pattern?
Then you won't need all those renames.

Reply:
These methods are not extension functions in Kotlin sense. However, I did some refactor and it should look better now.

Reply:
In particular. I cannot define the other class named `ProtocolTextDocument` since there is one generated already.